app/renderer/components/preferences/payment/enabledContent.js

```diff
@@ -462,7 +462,8 @@ class EnabledContent extends ImmutableComponent {
       <LedgerTable ledgerData={this.props.ledgerData}
         settings={this.props.settings}
         onChangeSetting={this.props.onChangeSetting}
-        siteSettings={this.props.siteSettings} />
+        siteSettings={this.props.siteSettings}
+        inProgress={ledgerData.get('status') === ledgerStatuses.IN_PROGRESS} />
```

Comment:
(Small) could we maybe bake this in to `this.props` so we have something like `this.props.paymentInProgress`?

Reply:
I can have it baked at this point.

Reply:
Sgtm, thanks!

Reply:
Done.